config/backend-config/backend-config.go

```diff
@@ -291,6 +288,33 @@ func (bc *CommonBackendConfig) WaitForConfig(ctx context.Context) {
 	}
 }
 
+func (bc *commonBackendConfig) SetUp() error        { return bc.workspaceConfig.SetUp() }
```

Comment:
we could get away with all these method declarations by embedding `workspaceConfig` interface in `commonBackendConfig` struct

```
type commonBackendConfig struct {
    workspaceConfig
    ....
}
```